frontend-zs/src/components/stories/StoryDetails.js

```diff
@@ -0,0 +1,47 @@
+
+import React from 'react';
+import { Row, Col, Media, Image } from 'react-bootstrap'
+import likeIcon from '../../icon_like.png'
+import './Story.scss'
+
+export default function StoryDetails(story) {
+    return (
+        <Row className="justify-content-between ml-3">
+            <Col className="mt-3">
+                <h2><strong>{story.title}</strong></h2>
+                {story.authorImage.length ? null : (
```

Comment:
Can we use the ``!!`` operator instead in these types of conditions? (The ``!!`` operator returns true for everything except the following: false, 0, empty string(""), null, undefined, and NaN)

```suggestion
                {!!story.authorImage.length ? (<h4 className="text-muted mb-2"><strong>{story.author}</strong></h4>) : null}

```

Reply:
I meant we should be dropping the ``.length`` parameter too. Sorry, my bad.